moveit_setup_assistant/src/widgets/robot_poses_widget.cpp

```diff
@@ -524,37 +514,37 @@ void RobotPosesWidget::loadJointSliders(const QString& selected)
 
   // Iterate through the joints
   int num_joints = 0;
-  for (std::vector<const robot_model::JointModel*>::const_iterator joint_it = joint_models_.begin();
-       joint_it < joint_models_.end(); ++joint_it)
+  for (const robot_model::JointModel* joint_model : joint_models_)
   {
-    // Check that this joint only represents 1 variable.
-    if ((*joint_it)->getVariableCount() == 1)
-    {
-      double init_value;
+    if (joint_model->getVariableCount() != 1 ||  // only consider 1-variable joints
+        joint_model->isPassive() ||              // ignore passive
+        joint_model->getMimic())                 // and mimic joints
+      continue;
 
-      // Decide what this joint's initial value is
-      if (joint_state_map_.find((*joint_it)->getName()) == joint_state_map_.end())
-      {
-        // the joint state map does not yet have an entry for this joint
+    double init_value;
 
-        // get the first joint value in its vector
-        (*joint_it)->getVariableDefaultPositions(&init_value);
-      }
-      else  // there is already a value in the map
-      {
-        init_value = joint_state_map_[(*joint_it)->getName()];
-      }
+    // Decide what this joint's initial value is
+    if (joint_state_map_.find(joint_model->getName()) == joint_state_map_.end())
+    {
+      // the joint state map does not yet have an entry for this joint
```

Comment:
This file uses capitalized comments (such as 3 lines above), please stay consistent

Reply:
These lower case comments were introduce by yourself ;-)
I will change them anyway.

Reply:
ah sorry :)